.github/CODEOWNERS

```diff
@@ -4,3 +4,4 @@
 /discovery/kubernetes @brancz
 /tsdb @codesome
 /promql @codesome @roidelapluie
+/cmd/promtool @jessicagreben @dgl
```

Comment:
@dgl you were wondering why notifications don't work, it turns out it was not configured.

Reply:
Thanks, that explains it :)